File: app/src/main/java/com/dhsdevelopments/potato/clientapi/users/LoadUsersResult.java
package com.dhsdevelopments.potato.clientapi.users;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class LoadUsersResult
{
    @SerializedName( "members" )
    public List<User> members;

    @Override
    public String toString() {
        return "LoadUsersResult[" +
                       "members=" + members +
                       ']';
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/users/LoadUsersResult.kt
package com.dhsdevelopments.potato.clientapi.users

import com.google.gson.annotations.SerializedName

class LoadUsersResult {
    @SerializedName("members")
    lateinit var members: List<User>

    override fun toString(): String {
        return "LoadUsersResult[members=$members]"
    }
}
